edgelet/Cargo.lock

```diff
@@ -1532,6 +1572,11 @@ dependencies = [
  "winapi 0.3.5 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "scoped-tls"
```

Comment:
This is new so it needs to be added to `THIRDPARTYNOTICES`. Though the only thing pulling this in is `tokio-core`, so when you remove that this should also go away.

Reply:
Yep, removed the dependency altogether